src/commands/install-kubeconfig.yml

```diff
@@ -0,0 +1,15 @@
+description: Install kubeconfig file
+
+parameters:
+  kubeconfig:
+    type: string
```

Comment:
Could this be of type `env_var_name` instead? (See: https://circleci.com/docs/2.0/reusing-config/#environment-variable-name) We recommend the `env_var_name` type for secrets to discourage users from supplying a secret as a literal string value